### PR TITLE
Reduce bundle size

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it reduces bundle size and slightly optimizes build time. It also adds bundle analyzing tools which are available by passing `ANALYZE=1` as an environment variable, like this:
```
$ ANALYZE=1 npm run build
```

**PR intended to be tested with API branch:** master

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -130,6 +129,8 @@
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^23.6.0",
     "babel-loader": "^8.0.6",
+    "babel-plugin-import": "^1.13.3",
+    "babel-plugin-lodash": "^3.3.4",
```
Comment (jwm0) [inline package.json:133]: These plugins are unused

Comment (dominik-zeglen) [reply package.json:133]: Right, cleanup

File: .eslintrc.json [hunk 2]
```diff
@@ -169,7 +169,13 @@
     "space-before-function-paren": "off",
     "spaced-comment": "error",
     "use-isnan": "error",
-    "valid-typeof": "off"
+    "valid-typeof": "off",
+    "no-restricted-imports": [
+      "error",
+      {
+        "paths": ["@material-ui/core", "lodash"]
```
[the remark this answers is not in the document]
Comment (dominik-zeglen) [reply .eslintrc.json:176]: Not really, it's optimized only for production build, but dev build has everything bundled in it, increasing build time - if ran only once it's not something developer would be concerned, but if one restarts it few times in a day (like I often do) it's a improvement. Lodash on the other hand does not support it at all.

**Review by orzechdev — APPROVED** [review]

**Review by mmarkusik — APPROVED** [review]

File: .eslintrc.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (jwm0) [reply .eslintrc.json:176]: As discussed internally we'll want to stick with root material-ui imports.

**Review by jwm0 — CHANGES_REQUESTED** [review]

**Review by jwm0 — APPROVED** [review]

**Pull request merged**